__tests__/app.test.js

```diff
@@ -69,7 +69,7 @@ describe("/api/reviews", () => {
     });
 });
 
-describe("/api/reviews/:review_id", () => {
+describe.only("/api/reviews/:review_id", () => {
```

Comment:
Please make sure that even when removing the .only, ALL tests pass before merging